priv/events/20190914161932_lonestar-elixir-2020.md

```diff
@@ -0,0 +1,36 @@
+{
+  "title": "Lonestar Elixir 2020",
+  "slug": "lonestar-elixir-2020",
+  "start": "2020-02-27T15:00:00.000000Z",
+  "end": "2020-02-29T23:00:00.000000Z",
+  "offset": "UTC-6",
+  "datetime": "2019-09-14T16:19:32.257433Z",
+  "event_url": "https://www.lonestarelixir.com/",
+  "gcal_url": "http://www.google.com/calendar/event?action=TEMPLATE&text=Lonestar%20Elixir&dates=20200227T090200Z/20200227T130200Z&https://www.lonestarelixir.com/&location=30.267153%2C-97.743061&trp=false&sprop=&sprop=name",
```

Comment:
do you use a tool to generate this url? If so, which one?

Reply:
I use `mix eef.gen.event` to generate a template, then a lot of manual copy/pasta :)